Dynamic-Programming/tests/LevenshteinDistance.test.js

```diff
@@ -0,0 +1,19 @@
+import { calculateLevenshteinDp } from '../LevenshteinDistance'
+
+test('Test Case 1', () => {
```

Comment:
Please use descriptive test case titles (or merge all tests).

Reply:
I just updated the titles, please let me know if they are fine